examples/sample-pipeline/sample-pipeline.cue

```diff
@@ -226,7 +220,7 @@ task: "kaniko": {
 			workingDir: "$(workspaces.source.path)"
 		}, {
 			name:  "write-url"
-			image: "bash"
+			image: "icr.io/gitsecure/bash"
```

Comment:
There needs to be an overall cleanup on the tasks and the images used. Based on SLSA we need to tie them to specific digest also: 
```
docker.io/library/bash:5.1.4@sha256:b208215a4655538be652b2769d82e576bc4d0a2bb132144c060efc5be8c3f5d6
```
Doing so will make sure that even if the tag changes, we still get the same version based on the digest that we trust.

Reply:
This is the format that the tekton catalog follows also. Just learned learned that based on the update I am making to the upstream kaniko task.

Reply:
tag and the digest. This way it will pin to both the tag and digest. So if the tag is overwritten upstream, we are still pulling the same digest we trusted before and not the new version of the tag.

Reply:
Note when you use both, only the digest is used, the tag becomes documentation for the user. I've seen people get confused thinking the tag is verified to point to that digest, where in reality the tag can claim anything it wants (and even be intentionally misleading).